source/routes/update/daily.js

```diff
@@ -0,0 +1,51 @@
+const PouchDB = require('pouchdb');
+PouchDB.plugin(require('pouchdb-find'));
+const db = new PouchDB('db');
+
+/* *
+ Update a daily entry's data
+ Request json must be in the form:
+ {
+    "_id": "documentID",
+    "updateField": {
+        "customFieldToUpdate": "customDataToUpdate",
+
+    }
+}
+*/
+module.exports = {
+  '/update/daily': {
+    methods: ['post'],
+    fn: function(req, res, next) {
+      // If the update request does not in form specified above, throw error
+      if (!req.body.updateField) {
+        throw new Error('MISSING UPDATE DATA');
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/cse110-sp21-group13/cse110-sp21-group13/pullRequest?prid=7433333)
```suggestion
        throw new Error("MISSING UPDATE DATA");
```